src-backend/api/tests/utils/decorators.py

```diff
@@ -0,0 +1,9 @@
+import api.startup
+
+# Use this decorator to add permissions to a new user
+# created in your setUp method for tests.
+def add_group_permissions(self):
+    def _run_once():
```

Comment:
This is just a single `_`, reason this was done was because this seems to be the convention for decorators.